src/index.html

```diff
@@ -7,12 +8,67 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="../src/styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
+
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="Apple A1419 iMac 27 Retina"
+      />
+
+      <h2 class="card__name">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <section class="card__stars">
+        <div class="stars stars--3">
```

Comment:
In this case, you have duplicate element `<div class="stars__star"></div>`. For cases with duplication, it is better to use a list, not div.

Reply:
done